core/Files.h

```diff
@@ -19,6 +19,7 @@ class FileRef final {
     FileRef() : _id(0){};
     FileRef(unsigned int id);
 
+    FileRef(FileRef &f) = default;
```

Comment:
Why is this needed?  Does something in the debugger not do proper resolution to select the `const &` overload?

Reply:
Yeah, I kept getting warnings in lldb until I added this.